packages/git/git.1.5.0/opam

```diff
@@ -45,7 +45,7 @@ depopts: [
 conflicts: [
  "cohttp"  {< "0.18.0"}
  "cohttp"  {>= "0.19.0"}
```

Comment:
I still cannot find the issue, I only found https://github.com/ocaml/opam-repository/issues/15441 which is somewhat unrelated. But look at the two lines above (and the same kind of repeated constraints are in many other packages)

Reply:
Tested locally and the `|` works fine